WcaOnRails/app/views/teams/edit.html.erb

```diff
@@ -0,0 +1,46 @@
+<% provide(:title, "Editing team #{@team.friendly_id}") %>
+
+<div class="container">
+
+  <h3><%= yield(:title) %></h3>
+
+  <%= simple_form_for @team, html: { class: 'form-horizontal are-you-sure no-submit-on-enter' }, wrapper: :horizontal_form,
+    wrapper_mappings: {
+      check_boxes: :horizontal_radio_and_checkboxes,
+      radio_buttons: :horizontal_radio_and_checkboxes,
+      file: :horizontal_file_input,
+      boolean: :horizontal_boolean
+    }  do |f| %>
+    <%= render 'shared/error_messages', object: f.object %>
+
+    <%= f.input :friendly_id, label: "Friendly id" %>
+    <%= f.input :name %>
+    <%= f.input :description %>
+
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        Members
+      </div>
+      <div class="panel-body">
+        <%= f.simple_fields_for :team_members do |t| %>
+          <%= t.input :user_id, class: 'form-control', required: true %>
```

Comment:
Those came from the team_member_fields file. They are probably not necessary here, indeed.